### ISSUE TITLE
Fail to parse RDF/XML from IdREF

### ISSUE THREAD

**Issue by pchampin** [oxigraph/oxigraph#1127]
**Describe the bug**
The (otherwise valid) RDF/XML produced by https://www.idref.fr/ raises an error in Oxigraph's parser.

**To Reproduce**
> Steps to reproduce the behavior:

```bash
curl -s https://www.idref.fr/074709267.rdf | oxigraph load -l $STORE --format rdf
```

> Which version of Oxigraph are you using? On which platform?

oxigraph 0.4.4 in Ubuntu Linux 24.10

The cause of the bug is that an unusual line break is represented by the XML entity `&#D;` (e.g. line 36 of https://www.idref.fr/074709267.rdf).

**Comment by Tpt** [comment]
Indeed, I forgot to update on of the code path, fixed in #1130

**Comment by Tpt** [comment]
The fix is shipped in v0.4.5. Feel free to reopen if it's not working properly.
